app/Http/Controllers/Admin/ModulesController.php

```diff
@@ -0,0 +1,93 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Contracts\Controller;
+use App\Models\Module;
+use App\Services\ModuleService;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\File;
+use Laracasts\Flash\Flash;
+
+class ModulesController extends Controller
+{
+    private $moduleSvc;
+
+    public function __construct(ModuleService $moduleSvc)
+    {
+        $this->moduleSvc = $moduleSvc;
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @param Request $request
+     *
+     * @return mixed
+     */
+    public function index()
+    {
+        $modules = Module::all();
+        return view('admin.modules.index', [
+            'modules' => $modules,
+        ]);
+    }
+
+    public function create()
+    {
+        return view('admin.modules.create');
+    }
+
+    public function store(Request $request)
+    {
+        $array = [];
+
+        array_push($array, $request->file('module_file'));
+        array_push($array, $request->has('enabled'));
```

Comment:
This is a little confusing/brittle, should look something like:

```php
$array = [
  'file'         => $request->file('module_file'),
  'enabled' => $request->get('enabled'),
];
```

That way we're not relying on indexes, and also, if there are other options added later on, then it's easy to update them.